controllers/article.controller.js

```diff
@@ -0,0 +1,21 @@
+const {allArticlesById, allArticles} = require('../models/articles.models')
+
+const getArticlesById = (request, response) => {
+    const article_id = parseInt(request.params.article_id)
+    console.log(request.params)
+         allArticlesById(article_id).then((article) => {
+            if(!article) {
+                return response.status(404).send({msg: 'Not Found'})
+            }
+        response.status(200).send({article: article})
+    })    
+    }
+   
+    const getArticles = (request, response) => {
+        allArticles().then((articleData) => {
+            response.status(200).send({articles: articleData})
+        })
+    }
```

Comment:
This is good, I like that you've chosen a nice simple key of `articles` to send the data back on